plugins/ModuleLevelTrigger.cpp

```diff
@@ -330,6 +344,16 @@ ModuleLevelTrigger::send_trigger_decisions()
   while (m_running_flag) {
     std::optional<triggeralgs::TriggerCandidate> tc = m_candidate_input->try_receive(std::chrono::milliseconds(10));
     if (tc.has_value()) {
+
+      if (m_first_tc) {
+        m_initial_offset = (duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count()) - (tc->time_start*(16*1e-6));
+        m_first_tc = false;
+      }
+
+      // Update OpMon Variable(s)
+      uint64_t system_time = duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count();
+      m_tc_data_vs_system.store(fabs(system_time - tc->time_start*(16*1e-6) - m_initial_offset));
```

Comment:
I guess the `16*1e-6` is not defined anywhere in the global configuration / appfwk / outside trigger where we could reasonably take it from? 
I imagine not, and if not, then maybe we should have a private `const` data member `m_` somewhere in the header with this number in (and a comment explaining what it is), and use that private data member instead of having magic numbers everywhere :/

Reply:
I'm not aware of this being anywhere in global config, but there are other occurrences of these translations elsewhere in the code. For now moved to header(s).